## 解題紀錄
### v1-1 RE
```
您的程式被監控系統中斷，可能是程式無法正常結束所導致。
Traceback (most recent call last):
  File "/14726011_d658/code_14726011.py", line 11, in 
    print(f'Case {i}: {math.ceil(math.log2(n))}')
ValueError: math domain error
```
> ??????

### v1-2 RE
```
(同v1-1)
```

### v1-3 RE
```
(同v1-1)
```

### v1-4 AC(45ms, 3.7MB)
- 不是到 -1 為止，而是到負數為止